themes/beaver/layouts/404.html

```diff
@@ -59,14 +59,9 @@ <h1 class="fl-heading">
                                         <div class="fl-photo fl-photo-align-center" itemscope
                                              itemtype="https://schema.org/ImageObject">
                                             <div class="fl-photo-content fl-photo-img-png">
-                                                <img loading="lazy" decoding="async"
-                                                     class="fl-photo-img wp-image-714 size-full"
-                                                     src="{{ absURL "wp-content/uploads/2022/10/404.png" }}"
-                                                alt="404" itemprop="image" height="750" width="802"
-                                                srcset="{{ "/wp-content/uploads/2022/10/404.png" }} 802w,
-                                                {{ "/wp-content/uploads/2022/10/404-300x281.png" }} 300w,
-                                                {{ "/wp-content/uploads/2022/10/404-768x718.png" }} 768w"
-                                                sizes="(max-width: 802px) 100vw, 802px" />
+
+                                                {{ $image := resources.Get "img/404.png" }}

```

Comment:
Can we provide a name to partial instead of `Get` here? This will simplify usage

Reply:
I use partial as a black box, so it doesn't know anything, and accept only image resource and title (for alt)

Reply:
Can we have partial to accept path to image and title

Reply:
@pftg no